lib/rubocop/cop/style/if_with_semicolon.rb

```diff
@@ -14,6 +14,7 @@ module Style
       #   result = some_condition ? something : another_thing
       #
       class IfWithSemicolon < Base
+        include IgnoredNode
```

Comment:
`IgnoredNode` is already included in `RuboCop::Cop::Base`, so including it again inside a cop is redundant.

Reply:
Ah, I see! I've updated this PR. Thank you.